packages/backend/src/tokens/clerkRequest.ts

```diff
@@ -62,7 +51,7 @@ class ClerkRequest extends Request {
 }
 
 export const createClerkRequest = (...args: ConstructorParameters<typeof ClerkRequest>): ClerkRequest => {
-  return new ClerkRequest(...args);
+  return args[0] instanceof ClerkRequest ? args[0] : new ClerkRequest(...args);
```

Comment:
Fight now, I'm using `ClerkRequest` directly in `clerk/nextjs` in order to calculate the correct URL for any redirects that need to take place (`redirectToSignIn` with `returnBackUrl` etc). Since I already have a `ClerkRequest` instance, I'm simply passing it in `authenticateRequest`, which normalizes `Request` to `ClerkRequest`.

I'd like us to move all redirect logic to clerk/backend but this needs to happen across all frameworks. For now, I'm simply avoiding initializing a second `ClerkRequest` with the `instanceof` check here.